### PR TITLE
Secret for signing tokens is retrived from AWS secrets manager

### PR DESCRIPTION
**Pull request by TeddyAndersson**

### CODE REVIEW

File: services/auth/token/helpers/token.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import jwt from 'jsonwebtoken';
 import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import secrets from '../../../../libs/secrets';
 
-// TODO: Retrive and use real SECRET_KEY from AWS;
-const SECRET_KEY = 'secretKeey';
+const SECRET_KEY = secrets.get('dev/token/generation', 'TokenGeneratorSecret');
```
Comment (dannil76) [inline services/auth/token/helpers/token.js:5]: Put in config.js

File: services/auth/token/lambdas/generateToken.js [hunk 2]
```diff
@@ -2,8 +2,7 @@ import { signToken } from '../helpers/token';
 
 export async function main(event) {
   const jsonRequest = JSON.parse(event.body);
-  const token = signToken(jsonRequest);
-
+  const token = await signToken(jsonRequest);
```
Comment (dannil76) [inline services/auth/token/lambdas/generateToken.js:5]: Await to?

File: services/auth/token/helpers/token.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 
-export function signToken(jsonToSign) {
-  const token = jwt.sign(jsonToSign, SECRET_KEY);
+export async function signToken(jsonToSign) {
+  const secret = await SECRET_KEY;
```
Comment (dannil76) [inline services/auth/token/helpers/token.js:8]: Await to?

File: libs/secrets.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import AWS from 'aws-sdk';
```
Comment (dannil76) [inline libs/secrets.js:1]: Should import our custom aws-sdk pkg to support future features

Comment (TeddyAndersson) [reply libs/secrets.js:1]: Don't se why we should use the aws-sdk pkg defined in /libs, it currently does nothing more than returning the aws import? Or for that manner why, we have added a wrapper somewhere along the line?

Comment (TeddyAndersson) [reply libs/secrets.js:1]: Just don't get the reason for having a wrapper, and i can see know that we have been using it mixed with normal aws-sdk imports. 

So some files are importing from 'aws-sdk' and others from './aws-sdk'

Comment (dannil76) [reply libs/secrets.js:1]: remove the wrapper for now then

**Pull request merged**